pkg/rancher-desktop/assets/dependencies.yaml

```diff
@@ -1,4 +1,4 @@
-limaAndQemu: "1.29"
+limaAndQemu: "1.28"
```

Comment:
Why are you downgrading Lima? This looks wrong.

Reply:
I downgraded Lima because I was getting an error when starting Rancher Desktop with `rdctl start`, as I was actually based on main before the bug fix of version 1.29, a rebase fixed the problem, so I am updating it back to 1.29